docs/toc.json

```diff
@@ -328,6 +328,7 @@
 					{ "docs/contributors/code/grammar.md": [] },
 					{ "docs/contributors/code/scripts.md": [] },
 					{ "docs/contributors/code/managing-packages.md": [] },
+					{ "docs/contributors/code/auto-cherry-picking.md": [] },
```

Comment:
Should we move it after the release section?

I see that React Native has even nested sections in the Block Editor Handbook. So maybe that instead?

Reply:
How about moving it _into_ the release section?

Reply:
Works for me 👍🏻